Dockerfile

```diff
@@ -5,9 +5,11 @@
 # In this file, we delete the *.ts intentionally
 # Any other changes to Dockerfile should be reflected in Publish
 
+
 # crane digest cgr.dev/chainguard/node-lts:latest-dev
 FROM cgr.dev/chainguard/node:latest-dev@sha256:0a7847d55e5837d4fd78cf1171ce09911e6b0a851bad27f44b8594ec101df9b6 AS build
 
+ENV HUSKY=0
```

Comment:
Is this necessary? I would have thought, `npm ci --omit=dev --omit=peer && \` would have omitted the need.